<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Cristian Munteanu</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1113</width>
    <height>840</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fordham Vehicle</string>
  </property>
  <property name="iconSize">
   <size>
    <width>46</width>
    <height>46</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Header">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1191</width>
      <height>841</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Hire">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>Hire</string>
     </attribute>
     <widget class="QLabel" name="headerLabel">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>0</y>
        <width>491</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hire a vehicle from Fordham Vehicle Hire</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="carTypeBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>170</y>
        <width>421</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <widget class="QRadioButton" name="btnSaloon">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Saloon</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="btnHP">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>High Performance</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="btnVan">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>141</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Van</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelSaloonPrice">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>20</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>£22.50</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelHPPrice">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>80</y>
         <width>72</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>£28.00</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelVanPrice">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>140</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>£35.00</string>
       </property>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>1191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>50</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>421</width>
         <height>21</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="resetBtn">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>660</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hireCostBtn">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>660</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Hire Cost</string>
      </property>
     </widget>
     <widget class="QLabel" name="selectLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>640</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Price</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="dayList">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>210</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>130</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Day/s</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>130</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Insurance Cover</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>600</y>
        <width>1191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>170</y>
        <width>20</width>
        <height>441</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>170</y>
        <width>20</width>
        <height>641</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>130</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Vehicle Type</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>1191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>340</y>
        <width>1191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_7">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>170</y>
        <width>16</width>
        <height>181</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>130</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>PPD</string>
      </property>
     </widget>
     <widget class="Line" name="line_10">
      <property name="geometry">
       <rect>
        <x>753</x>
        <y>170</y>
        <width>20</width>
        <height>181</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>830</x>
        <y>130</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Type of Customer</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="insuranceBox">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>170</y>
        <width>241</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="btnInsuranceYes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>95</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Yes </string>
       </property>
      </widget>
      <widget class="QRadioButton" name="btnInsuranceNo">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>60</y>
         <width>95</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>170</y>
        <width>351</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="btnCustomerNew">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>95</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>New</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="btnCustomerExisting">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>60</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Existing</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>350</y>
        <width>591</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QRadioButton" name="btnBronzePackage">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Bronze</string>
       </property>
      </widget>
      <widget class="QLabel" name="layalty">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>10</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Loyalty card type</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="btnSilverPackage">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>120</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Silver</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="btnGoldPackage">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>120</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Gold</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="Customers">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>Customers</string>
     </attribute>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>350</y>
        <width>421</width>
        <height>331</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Car selected</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="HiredCars">
     <attribute name="title">
      <string>Hired Cars</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>240</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>RadioButton</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
